pkg/scalers/azure/azure_cloud_environment.go

```diff
@@ -41,3 +45,7 @@ func ParseEnvironmentProperty(metadata map[string]string, propertyKey string, en
 	// Use public cloud suffix if `cloud` isn't specified
 	return envPropertyProvider(az.PublicCloud)
 }
+
+func ParseActiveDirectoryEndpoint(metadata map[string]string) (string, error) {
```

Comment:
Can we add some unit tests for this func please?

Reply:
Added.